operator/pkg/lbipam/lbipam_fixture_test.go

```diff
@@ -0,0 +1,293 @@
+package lbipam
+
+import (
+	"encoding/json"
+	"reflect"
+	"time"
+
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	cilium_api_v2alpha1 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
+	k8sClient "github.com/cilium/cilium/pkg/k8s/client"
+	cilium_fake "github.com/cilium/cilium/pkg/k8s/client/clientset/versioned/fake"
+	"github.com/cilium/cilium/pkg/k8s/client/clientset/versioned/typed/cilium.io/v2alpha1"
+	"github.com/cilium/cilium/pkg/k8s/resource"
+	"github.com/cilium/cilium/pkg/k8s/utils"
+	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	core_v1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/fake"
+	client_core_v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	k8s_testing "k8s.io/client-go/testing"
+	"k8s.io/client-go/tools/cache"
+)
+
+const (
+	poolAUID = types.UID("eb84f074-806e-474e-85c5-001f5780906b")
```

Comment:
What are these?

Reply:
Some pre-defined UUIDs which I use in the actual tests. Need them since I internally key on UUIDs, not including them in the objects would cause issues. This allows me to re-use a set of IDs so I don't have to copy them all the time